compute-aggregated.js

```diff
@@ -2,11 +2,26 @@ require("dotenv").config();
 const fs = require("fs");
 const { Elm } = require("./compute-aggregated-app");
 
+if (!process.env.ECOBALYSE_DATA_DIR) {
```

Comment:
See the remark made in server.js for the same check; it feels like some check code could be factored out.